### PR TITLE
Cleanup initial documentation

### PR DESCRIPTION
**Pull request by jarrpa**

### CODE REVIEW

**Comment by obnoxxx** [comment]
@jarrpa  sry. this needs rebase by now... ;-)

**Comment by obnoxxx** [comment]
spottet it too late

File: README.md [hunk 1]
```diff
@@ -9,11 +9,10 @@ deployment scheme.
...
-* **Glusterfs** (https://www.gluster.org/), the scale-out storage system, running as pods inside the kubernetes cluster.
+* **Kubernetes** (https://github.com/kubernetes/kubernetes/), the container management system.
+* **GlusterFS** (https://www.gluster.org/), the scale-out storage system, running as pods inside the kubernetes cluster.
 * **heketi** (https://github.com/heketi/heketi), Gluster's volume management service interface, running in a pod inside kubernetes.
 
-
 Resources:
 
-* [Documentation](./docs/index.md)
+* [Setup Guide](./docs/setup-guide.md)
```
Comment (obnoxxx) [inline README.md]: this hunk i do not agree with (without further arguments): I thought of having a slim little front page and having the people click further into documentation to select the doc they want to see, there. makes any sense at all?

**Comment by obnoxxx** [comment]
first patch is obsolete meanwhile

**Comment by jarrpa** [comment]
Updated the branch.

**Comment by obnoxxx** [comment]
I think you need to fetch and rebase once more ... sorry pal :-)

**Comment by jarrpa** [comment]
THERE. Once more, with feeling.

**Pull request merged**